features/keychain/module/crypto/secp256k1.js

```diff
@@ -6,9 +6,10 @@ export const create = ({ getPrivateHDKey }) => {
   const curve = new EC('secp256k1')
 
   const createInstance = () => ({
-    signBuffer: async ({ seedId, keyId, data }) => {
+    signBuffer: async ({ seedId, keyId, data, options }) => {
       const { privateKey } = getPrivateHDKey({ seedId, keyId })
-      return Buffer.from(curve.sign(data, privateKey).toDER())
+      const signature = curve.sign(data, privateKey, options)
+      return options?.rawSignature ? signature : Buffer.from(signature.toDER())
```

Comment:
prob better not mix return value `options` and options we pass through to `elliptic`. Can we whitelist both?

Reply:
How about having rawSignature as a separate param?

Reply:
sounds good, maybe `returnRawSignature`?

Reply:
although what is a raw signature? is it library specific or is it just 2 buffers?

Reply:
It's library specific. However we already rely on library specifics by calling `toDER()`.

Reply:
but `toDER()` is internal here, the caller doesn't have to know it's elliptic underneath. Why do we need to return the raw `elliptic` Signature instance?

Reply:
And even if it would be, when parsing using the `Signature` class we loose the `recovery` param value. So it isn't really usable.

Reply:
> but toDER() is internal here, the caller doesn't have to know it's elliptic underneath. Why do we need to return the raw elliptic Signature instance? (instead of some non-library-specific signature representation)

I can extract the 3 values that matter.

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/71/commits/be1cc8006200ff0eb564c879152ba5c3fc7606c0

Reply:
Maybe we can replace `rawSignature` with an `enc` option? The default value could be `der` and we'd specify `none` for the "raw" value?

Reply:
i like that idea, or maybe 'raw' instead of 'none'

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/71/commits/d7cae80720225a6dc05645843dbfbbaf2ccceed2